samples/drivers/flash_shell/boards/gd32vf103c_starter.conf

```diff
@@ -0,0 +1 @@
+CONFIG_HEAP_MEM_POOL_SIZE=8192
```

Comment:
nit: add a comment explaining why (same as explained in the commit message)

Reply:
Add a comment to this config, user can easily to know why this is needed.

Reply:
can't see the comment, missed push?

Reply:
Sorry, forget to apply this patch. done.